utils/rest_api/create_task.py

```diff
@@ -0,0 +1,259 @@
+            return res
+        status_resp_json = status_files_resp.json()
+        print("status_files_resp.json =")
+        pprint(status_resp_json)
+        if status_resp_json.get('state', "") in ("Finished", "Failed"):
+            break
+
+        time.sleep(1)
+
+    if status_resp_json.get('state', "") == "Finished":
+        print("Task is created and video is decoded for full_relative_path = '{}'".format(full_relative_path))
+    else:
+        print("ERROR DURING CREATION OF THE TASK '{}'".format(full_relative_path))
+        return res
+
+    print_hrule("BEGIN: Get Job Id")
+    job_id_resp = requests.get(cvat_uri + '/api/v1/tasks/{}'.format(task_id), verify=False, auth=(user, password))
+    print_hrule("END: Get Job Id")
+    if job_id_resp.status_code != 200:
+        print("CANNOT GET JOB ID, status code =", job_id_resp.status_code)
+        pprint(job_id_resp.json())
+        return res
+    job_id_json = job_id_resp.json()
+    pprint(job_id_json)
+    assert "segments" in job_id_json
+    segments = list(job_id_json["segments"])
+    assert segments
+    assert len(segments) == 1
+    assert "jobs" in segments[0]
+    jobs = segments[0]["jobs"]
+    assert len(jobs) == 1
+    job_id = jobs[0]["id"]
+    url_for_job = cvat_uri + "/?id={}".format(job_id)
+    print("url_for_job =", url_for_job)
+    res["url_for_job"] = url_for_job
+    res["status"] = True
+    return res
+
+def print_table_for_jira(list_all_done_res):
+    print_hrule("CURRENT RESULTS:")
+    for res in list_all_done_res:
+        assert res["status"]
+        print("|{}|[{}]|".format(res["full_relative_path"], res["url_for_job"]))
+    print_hrule("END OF CURRENT RESULTS")
+
+def main():
+    epilog_help_string = ("""
+    Please, note that argument path_json_labels_descr should point to a json file that describes labels for a task.
+    Example 1:
+        [
+            {
+                "name": "obj",
+                "attributes": []
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' without attributes,
+    it is equivalent to CVAT label string 'obj'
+
+    Example 2:
+        [
+            {
+                "name": "obj",
+                "attributes": [
+                                   {
+                                      "name": "class",
+                                      "mutable": false,
+                                      "input_type": "text",
+                                      "default_value": "",
+                                      "values": [""]
+                                   }
+                ]
+            }
+        ]
+    -- this will create for the task one bbox property with name 'obj' with one non-mutable text attribute 'class',
+    it is equivalent to CVAT label string'obj @text=class:""'
+    """)
+
+    parser = ArgumentParser(description = "Script that receives list of videos and to creates CVAT tasks (one for the video), "
```

Comment:
Very long text lines in many places. Could you please make the shorter (96 or even 80)?

Reply:
Fixed